applications/nrf_desktop/configuration/nrf54l15dk_nrf54l10_cpuapp/app_common.dtsi

```diff
@@ -6,7 +6,7 @@
 
 /* Application does not use cpuflpr core. Assign whole RRAM to cpuapp. */
 &cpuapp_rram {
-	reg = < 0x0 DT_SIZE_K(1022) >;
```

Comment:
In commit message:

 The NVM size
in the nRF54L15 SoC is equal to 1012 KB.

Shouldn't it be  The NVM size
in the nRF54L10 SoC is equal to 1012 KB.